.travis.yml

```diff
@@ -13,14 +13,11 @@ notifications:
       - bence.magyar.robotics@gmail.com
       - gm130s@gmail.com
 env:
-  global:
-    - CATKIN_PARALLEL_TEST_JOBS=-p1
-    - ROS_PARALLEL_TEST_JOBS=-j1
   matrix:
     - ROS_DISTRO=melodic  ROS_REPO=ros
     - ROS_DISTRO=melodic  ROS_REPO=ros-testing
-    - ROS_DISTRO=melodic  UPSTREAM_WORKSPACE=https://raw.githubusercontent.com/ros-controls/ros_control/$ROS_DISTRO-devel/ros_control.rosinstall
+    - ROS_DISTRO=melodic  UPSTREAM_WORKSPACE='https://raw.githubusercontent.com/ros-controls/ros_control/$ROS_DISTRO-devel/ros_control.rosinstall -ros_controllers'
```

Comment:
Is there a reason we have this `rosinstall` setup as a third job, rather than applied to both other jobs (As is done in all the other `ros_control` repos?) I think this actually might make more sense than the other repos, since we're testing against 3 different versions of the other `ros_control` packages rather than always pulling from GitHub.

Reply:
I think the reason for this was to allow breakage in one direction but not in others.